src/java.net.http/share/classes/jdk/internal/net/http/common/Utils.java

```diff
@@ -367,10 +367,32 @@ public static String describeOps(int interestOps) {
     public static IllegalArgumentException newIAE(String message, Object... args) {
         return new IllegalArgumentException(format(message, args));
     }
+
+    /**
+     * {@return a new {@link ByteBuffer} instance of configured capacity for the HTTP Client}
+     */
     public static ByteBuffer getBuffer() {
         return ByteBuffer.allocate(BUFSIZE);
     }
 
+    /**
+     * {@return a new {@link ByteBuffer} instance whose capacity is set to the
+     * smaller of the specified {@code maxCapacity} and the default
+     * ({@value BUFSIZE})}
+     *
+     * @param maxCapacity a buffer capacity, in bytes
+     * @throws IllegalArgumentException if {@code capacity < 0}
```

Comment:
Typo - should have been "if {@code maxCapacity < 0}"

Reply:
Fixed in da69840639e.